### DISCUSSION TITLE
Migrate from NIPAP

### DISCUSSION THREAD

**Discussion by MrPaulAR** [netbox-community/netbox#9423] [Q&A]
Is there a mechanism to migrate from NIPAP?

**Comment by matejv** [comment]
This kind of migration is waiting for us ass wall in the next months. As far as I'm aware NIPAP web interface lacks some sort of export functionality. But there is an [example python script](https://github.com/SpriteLink/NIPAP/blob/master/utilities/export-csv.py) that uses the XML-RPC API to get data and write it to a CSV.

You could then manipulate that data in Excel to get a format suitable for importing into netbox.
